src/Symfony/Component/Mailer/Transport/AbstractApiTransport.php

```diff
@@ -37,6 +37,9 @@ protected function doSendHttp(SentMessage $message): ResponseInterface
         return $this->doSendApi($message, $email, $message->getEnvelope());
     }
 
+    /**
+     * @return array<Address>
```

Comment:
Let's keep it simple and consistent with the phpdoc in Envelope. 

```suggestion
     * @return Address[]
```